pkg/broadcaster/unbuffered.go

```diff
@@ -1,18 +1,18 @@
-package broadcastwriter
+package broadcaster
 
 import (
 	"io"
 	"sync"
 )
 
-// BroadcastWriter accumulate multiple io.WriteCloser by stream.
-type BroadcastWriter struct {
+// Unbuffered accumulate multiple io.WriteCloser by stream.
+type Unbuffered struct {
 	mu      sync.Mutex
 	writers []io.WriteCloser
 }
 
 // AddWriter adds new io.WriteCloser.
-func (w *BroadcastWriter) AddWriter(writer io.WriteCloser) {
+func (w *Unbuffered) AddWriter(writer io.WriteCloser) {
```

Comment:
I think I'd rather not. If the writers we're passing in have always have a `Close` method and we call that `Close` method, I think it's better to type them as `io.WriteClosers`. The difference between the buffered and unbuffered versions isn't ideal, but I'd rather have that different plainly exposed than disguise it.